test-integration/samples/.jhipster/BankAccount.json

```diff
@@ -38,6 +38,10 @@
             "fieldName": "meanOperationDuration",
             "fieldType": "Double"
         },
+        {
+            "fieldName": "meanQueueDuration",
```

Comment:
It would be better to add a field of type Duration in FieldTestEntity.json : it's used for nearly all database types, and already contains every field type that JHipster supports, so it would be incomplete without Duration ;)

Reply:
Hi @murdos, I added it to the list of fields. Will the CI run the relative tests?

Reply:
Thanks! Yes, it will.